Lib/test/test_ssl.py

```diff
@@ -28,6 +28,9 @@
 
 from ssl import TLSVersion, _TLSContentType, _TLSMessageType, _TLSAlertType
 
+Py_DEBUG = hasattr(sys, 'gettotalrefcount')
+Py_DEBUG_Win32 = Py_DEBUG and sys.platform=='win32'
```

Comment:
```suggestion
Py_DEBUG_WIN32 = Py_DEBUG and sys.platform == 'win32'
```

I'll let you update the references to this variable.